build.gradle

```diff
@@ -86,7 +87,15 @@ dependencies {
         exclude group: 'io.netty', module: 'netty-transport-native-unix-common'
     })
     implementation group: 'com.aliyun', name: 'alidns20150109', version: '3.0.1'
-    implementation group: 'xerces', name: 'xercesImpl', version: '2.11.0'
+    implementation group: 'xerces', name: 'xercesImpl', version: '2.12.2'
```

Comment:
It is recommended to remove packages that are confirmed to be useless.

Reply:
The package xercesImpl has been deleted.